continuous_integration/appveyor/environment.yaml

```diff
@@ -1,10 +1,11 @@
 name: testenv
 channels:
   - conda-forge
+  - defaults
```

Comment:
Do we need the default channel, or does `conda-forge` suffice?

Reply:
This looks correct to me. Dropping it would mean we don't use `defaults` at all, which is probably not what we want.

Reply:
Is there a reason to use `defaults` if all the packages we want to install are in `conda-forge`?

Reply:
> Dropping it would mean we don't use defaults at all, which is probably not what we want.

Actually, IIUC this change was worthless except for explicitly orderining `conda-forge` and `defaults` - we'd have to add `nodefaults` to the channel list to disable defaults entirely (otherwise its implicitly there): https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html

The real fix here was to pin numpy, but I'm reluctant to make another commit since this PR works as is and this change is harmless.

Reply:
If so, we may consider adding `defaults` to our [other CI environments](https://github.com/dask/dask/blob/35166b3896d9169f0d0fbf71745512b862ae4638/continuous_integration/travis/travis-37.yaml) (doesn't have to be in this PR)

Reply:
> I'm reluctant to make another commit since this PR works as is and this change is harmless

Totally fine by me. I was just curious if I was missing something about how specifying channels works with `conda`